src/game/PlayerBots/PlayerBotMgr.cpp

```diff
@@ -1763,6 +1762,28 @@ bool ChatHandler::HandleBattleBotRemoveAllCommand(char* args)
     return true;
 }
 
+bool ChatHandler::HandleBattleBotAutoJoinCommand(char* args)
+{
+    if (!*args)
+    {
+        SendSysMessage(m_confBattleBotAutoJoin ? "Battlebots AutoJoin - on" : "Battlebots AutoJoin - off");
+        return true;
+
+    }
+
+    bool value;
+    if (!ExtractOnOff(&args, value))
+    {
+        SendSysMessage(LANG_USE_BOL);
+        SetSentErrorMessage(true);
+        return false;
+    }
+
+    m_confBattleBotAutoJoin = value ? true : false;
```

Comment:
Same issue here as above.

Reply:
Clearly, the problem is with the scope, how is it fixed?